spec/lib/metasploit/cache/module/class/ephemeral/rank_spec.rb

```diff
@@ -0,0 +1,128 @@
+RSpec.describe Metasploit::Cache::Module::Class::Ephemeral::Rank do
+  context '#synchronize' do
+    include_context 'ActiveSupport::TaggedLogging'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.